nightswatcher/nightswatcher.py

```diff
@@ -174,10 +177,29 @@ def extract_build(artifact_zip, build):
     tmp_artifact_path = tmp_version_dir + download_artifact
     if build['name'].startswith('build_android'):
         sign_apk(tmp_artifact_path)
-    shutil.move(tmp_artifact_path, download_artifact_path)
+    shutil.copy2(tmp_artifact_path, download_artifact_path)
+
+    ota_artifact_path = tmp_version_dir + download_artifact
+    # point update pointer to the right location
+    if build['name'] in ota_link_models:
+        if build['name'] == 'build_android_x86':
+            platform = platform + '-x86'
+
+        link_file_stable = OTA_DIR + ('koreader-%s-latest-stable' % platform)
+        link_file_nightly = OTA_DIR + ('koreader-%s-latest-nightly' % platform)
+        link_file = stable is True and link_file_stable or link_file_nightly
+
+        os.mklink(download_artifact_path, OTA_DIR + download_artifact)
```

Comment:
Hm, this must be newer or something? I'll investigate later.

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/gevent/greenlet.py", line 534, in run
    result = self._run(*self.args, **self.kwargs)
  File "/nightswatcher.py", line 236, in fetch_build
    extract_build(artifact_zip, build)
  File "/nightswatcher.py", line 192, in extract_build
    os.mklink(download_artifact_path, OTA_DIR + download_artifact)
AttributeError: 'module' object has no attribute 'mklink'
```

Reply:
D'oh, the method is called `symlink`. My brain was operating on "it's not `ln`…" but incorrectly ended up at the Windows version. :-D